### DISCUSSION TITLE
Upgrade to Capacitor 3 and no bridge found

### DISCUSSION THREAD

**Discussion by stevenroussey-privicy** [ionic-team/capacitor#5200] [Q&A]
I followed the direction in the upgrade guide and now at the point to run things in the iOS simulator. But the app crashes on startup:

```
ERROR: Unable to find application directory at: "file:///Users/steve/Library/Developer/CoreSimulator/Devices/B0B74518-4D55-470B-8BDB-0232A3AF0AF6/data/Containers/Bundle/Application/24FB71A5-A7B1-4BA7-910B-CB37F4D657E2/App.app/public"!
2021-11-02 22:01:09.013274-0700 App[50923:5813399] Writing analzed variants.
ERROR: Required cordova.js file not found. Cordova plugins will not function!
⚡️ ❌  Capacitor: FATAL ERROR
⚡️ ❌  Error was:  Unable to export JavaScript bridge code to webview
⚡️ ❌  Unable to export required Bridge JavaScript. Bridge will not function.
⚡️ ❌  You should run "npx capacitor copy" to ensure the Bridge JS is added to your project.
⚡️ ❌  Please verify your installation or file an issue
⚡️  ERROR: Unable to load /Users/steve/Library/Developer/CoreSimulator/Devices/B0B74518-4D55-470B-8BDB-0232A3AF0AF6/data/Containers/Bundle/Application/24FB71A5-A7B1-4BA7-910B-CB37F4D657E2/App.app/public
⚡️  This file is the root of your web app and must exist before
⚡️  Capacitor can run. Ensure you've run capacitor copy at least
⚡️  or, if embedding, that this directory exists as a resource directory.
```

The upgrade guide is dated for the current version of xcode. There it said to delete the public folder and re-create it using the left-click, there is no "Add folder" but there was "Add Group" which seemed to do the same thing. Do you think that is the issue?

**Comment by jcesarmobile** [comment]
Add Group doesn't do the same thing than Add folder, so that's probably the issue.

**Comment by stevenroussey-privicy** [comment]
![image](https://user-images.githubusercontent.com/42726156/141194246-9361b338-a46d-4792-a506-947dea8877d3.png)

OK, so what do I do instead? The instructions on the website are wrong/old and do not match reality of xcode today.

**Reply by jcesarmobile** [reply]
The guide says 
> Right-click on the App group inside the App project and click Add Files to “App”…

I can see the Add Files to “App” in your screenshot
